pkg/collector/corechecks/containers/cri.go

```diff
@@ -0,0 +1,110 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2018 Datadog, Inc.
+
+// +build cri
+
+package containers
+
+import (
+	yaml "gopkg.in/yaml.v2"
+	pb "k8s.io/kubernetes/pkg/kubelet/apis/cri/runtime/v1alpha2"
+
+	"github.com/DataDog/datadog-agent/pkg/aggregator"
+	"github.com/DataDog/datadog-agent/pkg/autodiscovery/integration"
+	"github.com/DataDog/datadog-agent/pkg/collector/check"
+	core "github.com/DataDog/datadog-agent/pkg/collector/corechecks"
+	"github.com/DataDog/datadog-agent/pkg/tagger"
+	"github.com/DataDog/datadog-agent/pkg/util/containers"
+	"github.com/DataDog/datadog-agent/pkg/util/containers/cri"
+	"github.com/DataDog/datadog-agent/pkg/util/log"
+)
+
+const (
+	criCheckName = "cri"
+)
+
+// CRIConfig holds the config of the check
+type CRIConfig struct {
+	Tags        []string `yaml:"tags"`
+	CollectDisk bool     `yaml:"collect_disk"`
+}
+
+// CRICheck grabs CRI metrics
+type CRICheck struct {
+	core.CheckBase
+	instance *CRIConfig
+}
+
+func init() {
+	core.RegisterCheck("cri", CRIFactory)
+}
+
+// CRIFactory is exported for integration testing
+func CRIFactory() check.Check {
+	return &CRICheck{
+		CheckBase: core.NewCheckBase(criCheckName),
+		instance:  &CRIConfig{},
+	}
+}
+
+// Parse parses the CRICheck config and set default values
+func (c *CRIConfig) Parse(data []byte) error {
+	// default values
+	c.CollectDisk = false
+
+	if err := yaml.Unmarshal(data, c); err != nil {
+		return err
+	}
+	return nil
+}
+
+// Configure parses the check configuration and init the check
+func (c *CRICheck) Configure(config, initConfig integration.Data) error {
+	return c.instance.Parse(config)
```

Comment:
Can you please add a call to [CommonConfigure](https://github.com/DataDog/datadog-agent/blob/master/pkg/collector/corechecks/checkbase.go#L65)

Reply:
Of course 🤦‍♂️